osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/README.md

```diff
@@ -82,4 +82,51 @@ A test request could look like this:
 </soapenv:Envelope>
 ```
 
-The RtuDevice will be inserted in the osgp-core database and picked up by the communication monitoring task.
\ No newline at end of file
+The RtuDevice will be inserted in the osgp-core database and picked up by the communication monitoring task.
+
+## Add MQTT RTU Device in the DB
+Alternative way to add RTU device.
+
+```
+INSERT INTO public.protocol_info(
+            creation_time, modification_time, version, protocol, protocol_version, 
+            outgoing_requests_property_prefix, incoming_responses_property_prefix, 
+            incoming_requests_property_prefix, outgoing_responses_property_prefix, 
+            parallel_requests_allowed)
+    VALUES ('2020-02-24 00:00:00', '2020-02-24 00:00:00', 0, 'MQTT', '3', 
+    'jms.protocol.mqtt.outgoing.requests', 'jms.protocol.mqtt.incoming.responses',
+    'jms.protocol.mqtt.incoming.requests', 'jms.protocol.mqtt.outgoing.responses',TRUE
+    );
+```
+protocol_info.id is auto generated
+
+```    
+INSERT INTO public.device(
+            creation_time, modification_time, version, device_identification, 
+            device_type, is_activated,                 
+            protocol_info_id, network_address,  
+            in_maintenance, technical_installation_date, 
+            device_model, device_lifecycle_status)
+    VALUES ('2020-01-01 12:34', '2020-01-01 12:34', 0, 'TST-1', 
```

Comment:
I would prefer something like NOW() over a fixed datetime. 
Same applies to SQL statements below.